include/libtorrent/alert_types.hpp

```diff
@@ -56,6 +56,9 @@ POSSIBILITY OF SUCH DAMAGE.
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #endif
 
+// c++/cx keyword
+#undef interface
```

Comment:
not bc alert_types.hpp is pulled in by stuff qtorrent use. session_status (now stats_metric) and perhaps other stuff. better to hide it from winrt so it can use some of that old stuff